WISE-Community/WISE-Client
# feat(Tag): Prevent duplicate tags when editing tag

## Changes
- Created EditTagDialogComponent
- Created AbstractTagDialogComponent that CreateTagDialogComponent and EditTagDialogComponent both extend
- Prevent duplicate tags when editing a tag

## Test
- Test with https://github.com/WISE-Community/WISE-API/pull/269
- Make sure you can't create a duplicate tag when editing a tag
- Make sure you can't create a duplicate tag when creating a new tag
